pensjonsmaler/src/main/kotlin/no/nav/pensjon/brev/maler/redigerbar/TilbakekrevingAvFeilutbetaltBeloep.kt

```diff
@@ -0,0 +1,208 @@
+                        English to "In the same period, you have received too little "
+                    )
+                    includePhrase(Redigerbar.SaksType(pesysData.sakstype))
+                    text(
+                        Bokmal to "Dette er tatt med i beregningen vår.",
+                        Nynorsk to "Dette er teke med i berekninga vår.",
+                        English to "This has been taken into account in our calculations."
+                    )
+                }
+            }
+            paragraph {
+                text(
+                    Bokmal to "Vedtaket er gjort etter folketrygdloven § 22-15.",
+                    Nynorsk to "Vedtaket er gjort etter folketrygdlova § 22-15.",
+                    English to "This decision is made pursuant to the provisions of section 22-15 of the National Insurance Act."
+                )
+            }
+            paragraph {
+                text(
+                    Bokmal to "I vedlegget “Oversikt over feilutbetalinger” finner du en oversikt over periodene med feilutbetalinger og beløpet du må betale tilbake.",
+                    Nynorsk to "I vedlegget “Oversikt over feilutbetalinger” finn du ei oversikt over periodane med feilutbetalingar og beløpet du må betale tilbake.",
+                    English to "The attachment titled 'Overview of Incorrect Payments' provides details on the periods with payment errors and the amounts that need to be repaid."
+                )
+            }
+            title1 {
+                text(
+                    Bokmal to "Betydning for skatteoppgjøret",
+                    Nynorsk to "Betydning for skatteoppgjeret",
+                    English to "Significance for the tax settlement"
+                )
+            }
+            paragraph {
+                text(
+                    Bokmal to "Vi rapporterer endringen til Skatteetaten. De vil korrigere skatteoppgjøret ditt ut fra denne endringen.",
+                    Nynorsk to "Vi rapporterer endringa til Skatteetaten. Dei vil korrigere skatteoppgjeret ditt ut frå denne endringa.",
+                    English to "We will report the change to the Norwegian Tax Administration. They will correct your tax settlement in view of this change.",
+                )
+            }
+            title1 {
+                text(
+                    Bokmal to "Du har rett til å klage",
+                    Nynorsk to "Du har rett til å klage",
+                    English to "You have the right to appeal"
+                )
+            }
+            paragraph {
+                text(
+                    Bokmal to "Hvis du mener vedtaket er feil, kan du klage innen seks uker fra den datoen du mottok vedtaket. " +
+                            "Klagen skal være skriftlig. Du finner skjema og informasjon på nav.no/klage. I vedlegget “Dine rettigheter og mulighet til å klage” får du vite mer om hvordan du går fram.",
```

Comment:
Bruk named reference for vedlegg. Vi har også brukt andre quotation marks andre steder: «». Ser vi har lignende fraser, men de refererer ikke til vedlegget.